faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+## How do I restart my JupyterLab server?
+
+On Libretexts JupyterHub, you spawn servers that provide you with JupyterLab software and you execute your code using kernels. If you are actively running code and something unexpected occurs, [restarting the kernel](https://jupyterlab.readthedocs.io/en/stable/user/running.html) can often be an easy solution. However, sometimes the problem may be deeper than that, especially if some of your files were incorrectly modified. In this case, it could be a good idea to restart your server. 
+
+Go to File->Hub Control Panel and you will be brought to a page with "Stop Server" and "My Server" buttons. Press Stop Server and you should see that that the My Server button will dim; this means that the server is stopping. When the server has stopped (this should take less than 15 seconds), the webpage will prompt you with a "Start My Server" option. Press it, launch the server, and then you will be brought back to the Server Options page. Proceed with the environment of your choice, and you will now have a fresh server.
+
+By default, your server will always shutdown after 1 hour of inactivity. All files which you wish to modify and save across restarts must be located in your `/home/jovyan` directory; everything else will be wiped. Think of these user files as being stored in a cloud, and each time you start JupyterLab, we provide you with a brand new server which has downloaded your specific files. For more technical information on how your files are stored, see the Jupyterhub for Kubernetes documentation [here](https://zero-to-jupyterhub.readthedocs.io/en/latest/customizing/user-environment.html#about-user-storage-and-adding-files-to-it). 
+
+## Can I recieve accouncements for downtime and other critical information about jupyter.libretexts.org?
+
+If you wish to recieve updates from the JupyterTeam about possible downtime for jupyter.libretexts.org, you can subscribe to our [mailing list](https://lists.ucdavis.edu/sympa/subscribe/flock-announce). Just enter your email and you will recieve emails whenever we notify Hub users about downtime and maintenance.
+
+# For Instructors
+
+## How do I distribute files to students?
+
+You can distribute files using [nbgitpuller](https://jupyterhub.github.io/nbgitpuller/index.html). Please store your files in a GitHub repository and we can set up a custom environment that will update the files whenever a student spawns a server. This is a good option since students don't need to understand git, but keep in mind that nbgitpuller does have [some limitations](https://jupyterhub.github.io/nbgitpuller/topic/automatic-merging.html). Other options include hosting your files on an external site, like GitHub or Google Drive, where students can download them.
+
+## How do I set up custom environments for my class?
```

Comment:
I think we should probably remove this section for now. When we get someone that default-env can't support, then we can figure out the best way to do this. In the future we can support advanced users that create custom repo2docker configuration files, but for now, no.

Reply:
Should we just remove the whole "For Instructors" section then? It's simply copy pasted from the previous FAQ page. For the most part, the content within that section is already covered elsewhere, it just isn't specifically targeted at instructors.

Reply:
No, remove "## How do I set up custom environments for my class?"